README.md

```diff
@@ -45,6 +46,30 @@ If your IDE supports it, you can directly run the .feature files within `/featur
 Run the tests by running `behave` in a command prompt or terminal window.
 * This will run the tests and print the results to console
 
+```
+behave -D product=EPS-FHIR -D env=INT -f behave_cucumber_formatter:PrettyCucumberJSONFormatter -o reports/cucumber_json.json -f
+allure_behave.formatter:AllureFormatter -o allure-results -f pretty features --no-capture --no-capture-stderr --no-skipped --expand --logging-level=DEBUG --tags eps_fhir
+```
+
+change the `env` variable accordingly to either `INT` or `INTERNAL-DEV`
+If you with to test a different product i.e. `PFP-APIGREE` then you must change `product=` and `--tags` respectively
+
 ### Setting the BASE_URL
-To run the tests from your IDE it is necessary to set the BASE_URL environment variable.
-<br /> If running via GitHub actions this will default to `INT` and currently cannot be changed
+The BASE_URL is set based on the environment you provide in the above command. This cannot be overridden
+
+
+### Getting the token to check the endpoint calls on Postman
+
+Depending on which environment you have run your *behave* command you can pick from the following:
+
+- INTERNAL-DEV: Call `https://sxhjsbv4d7tvmt67av3jlboera0yzvgc.lambda-url.eu-west-2.on.aws/?env=internal-dev` which provides the Authorization token for the internal-dev environment.
```

Comment:
Could work the internal dev call into the get_token script for consistency here.

Reply:
Comment addressed.